scripts/powershell/ref-updater.ps1

```diff
@@ -0,0 +1,57 @@
+# Updates the Ref in all Yml files in the Repository
+param (
+    [String]$BuildSourcesDirectory
+)
+
+Install-Module -Name powershell-yaml -RequiredVersion 0.4.2 -Force -Scope CurrentUser
+$Repos = @('azure-sdk-for-net', 'azure-sdk-for-js', 'azure-sdk-for-java', 'azure-sdk-for-python')
+
+# Get Latest tag
+$CurrentDate = Get-Date -Format "yyyyMMdd"
+pushd "$BuildSourcesDirectory\azure-sdk-tools"
+$toolsRepoTags = Sort-Object -InputObject ([String[]](git tag -l "azure-sdk-tools_$CurrentDate.*")) -Descending
+$toolsRepoLatestTag = $toolsRepoTags[0]
+popd
+pushd "$BuildSourcesDirectory\azure-sdk-build-tools"
+$buildToolsRepoTags = Sort-Object -InputObject ([String[]](git tag -l "azure-sdk-build-tools_$CurrentDate.*")) -Descending
+$buildToolsRepoLatestTag = $buildToolsRepoTags[0]
+popd
```

Comment:
I think I would remove this from the script and have the script take the repo and tag as parameters. I expect we will not run it by taking the latest we will istead run it as part of the release process for the particular tools release process and so we will have the tag.